ctapipe/ml/tests/test_sklearn.py

```diff
@@ -0,0 +1,139 @@
+import pytest
+
+import numpy as np
+from traitlets.config import Config
+from traitlets import TraitError
+from astropy.table import Table
+from numpy.testing import assert_array_equal
+
+
+def test_supported_regressors():
```

Comment:
If the whitelist approach is to be used for the list of supported models, there should be a test that an unsupported model is not in the `SUPPORTED_REGRESSORS`, e.g. the DummyRegressor